Makefile

```diff
@@ -55,26 +48,24 @@ build-container: require-GCE_PD_CSI_STAGING_IMAGE require-GCE_PD_CSI_STAGING_VER
 		--build-arg STAGINGVERSION=$(STAGINGVERSION) \
 	  --push .
 
-build-and-push-windows-container-ltsc2019: require-GCE_PD_CSI_STAGING_IMAGE init-buildx
-	$(DOCKER) buildx build --file=Dockerfile.Windows --platform=windows \
-		-t $(STAGINGIMAGE):$(STAGINGVERSION)_ltsc2019 \
-		--build-arg BASE_IMAGE=$(BASE_IMAGE_LTSC2019) \
-		--build-arg STAGINGVERSION=$(STAGINGVERSION) --push .
+# TODO(alexander-ding): the Windows image built here should work on every Windows Server version
+# however, due to https://github.com/containerd/containerd/issues/7431 we still need to tag
+# a specific Windows OS version in the image. For now, we tag Windows ltsc2019, but after the issue
+# is resolved, we can remove the docker manifest annotate lines and rename the manifest to $(STAGINGVERSION)_windows
 
-build-and-push-windows-container-20H2: require-GCE_PD_CSI_STAGING_IMAGE init-buildx
-	$(DOCKER) buildx build --file=Dockerfile.Windows --platform=windows \
-		-t $(STAGINGIMAGE):$(STAGINGVERSION)_20H2 \
-		--build-arg BASE_IMAGE=$(BASE_IMAGE_20H2) \
+build-and-push-container-windows-ltsc2019: require-GCE_PD_CSI_STAGING_IMAGE init-buildx
+	$(DOCKER) buildx build --file=Dockerfile.Windows --platform=windows/amd64 \
+		-t $(STAGINGIMAGE):$(STAGINGVERSION)_windows_ltsc2019 \
 		--build-arg STAGINGVERSION=$(STAGINGVERSION) --push .
 
-build-and-push-multi-arch: build-and-push-container-linux-amd64 build-and-push-container-linux-arm64 build-and-push-windows-container-ltsc2019 build-and-push-windows-container-20H2
-	$(DOCKER) manifest create --amend $(STAGINGIMAGE):$(STAGINGVERSION) $(STAGINGIMAGE):$(STAGINGVERSION)_linux_amd64 $(STAGINGIMAGE):$(STAGINGVERSION)_linux_arm64 $(STAGINGIMAGE):$(STAGINGVERSION)_20H2 $(STAGINGIMAGE):$(STAGINGVERSION)_ltsc2019
-	STAGINGIMAGE="$(STAGINGIMAGE)" STAGINGVERSION="$(STAGINGVERSION)" WINDOWS_IMAGE_TAGS="$(WINDOWS_IMAGE_TAGS)" WINDOWS_BASE_IMAGES="$(WINDOWS_BASE_IMAGES)" ./manifest_osversion.sh
+build-and-push-multi-arch: build-and-push-container-linux-amd64 build-and-push-container-linux-arm64 build-and-push-container-windows-ltsc2019
+	$(DOCKER) manifest create --amend $(STAGINGIMAGE):$(STAGINGVERSION) $(STAGINGIMAGE):$(STAGINGVERSION)_linux_amd64 $(STAGINGIMAGE):$(STAGINGVERSION)_linux_arm64 $(STAGINGIMAGE):$(STAGINGVERSION)_windows_ltsc2019
+	$(DOCKER) manifest annotate --os-version 10.0.17763 $(STAGINGIMAGE):$(STAGINGVERSION) $(STAGINGIMAGE):$(STAGINGVERSION)_windows_ltsc2019
```

Comment:
A comment saying how you got `10.0.17763` will be useful if we need to update this later